src/scss/base/_typography.scss

```diff
@@ -24,24 +24,32 @@ html {
 }
 
 body {
-    font-family: $font-family-base;
+    font-family: $font-family-preload;
     @include font-size(base);
     color: $color-text;
     font-weight: $font-weight-base;
 
     text-rendering: optimizelegibility;
     -webkit-font-smoothing: antialiased;
     -moz-font-smoothing: antialiased;
+
+    .has-fontsLoaded--base & {
```

Comment:
I think I'd rather this work the other way around to how it is atm, as it'll be easier to maintain going forward (as we'd have to add this modifier everywhere a font-family is defined doing this).

Instead, we could apply the `$font-family-preload;` to a set of selectors:

```
.is-fontsLoading {
  body,
  p,
  h1… {
    font-family: $font-family-preload !important;
  }
}
```

and remove the `is-fontsLoading` class once they're loaded which will return the styles to the default as defined.